src/settings/menuoptions.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * The purpose of this class is to show widget with application' settings.
+ * Some parametrs just for preview, some for change.
+ * There are 5 tabs:
+ *  Environment - get access to configuration file; changing application style.
+ *  Text editor - with editor' settings: font size, font name.
+ *  C++ - with compiler' paths; C++ standart.
+ *  Repository - not implemented yet
+ *  Connection - with UDP, TCP numbers
+*/
+#ifndef MENUOPTIONS_H
+#define MENUOPTIONS_H
+
+#include "tabenvironment.h"
+#include "tabtexteditor.h"
+#include "tabconnection.h"
+#include "tabcpp.h"
+#include "tabrepository.h"
+
```

Comment:
Please move headers in lines 14-18 to cpp file
Add forward declarations instead

Reply:
removed